operator/README.md

```diff
@@ -0,0 +1,6 @@
+# Metabroker Operator
+
+The Metabroker Operator is responsible for managing the CRDs for:
+- Service Providers
+- Service Offerings
+- Service Plans
```

Comment:
Please mention `kubebuilder` in here, so that we don't need to go to the commit history to find it. No need to mention the commands used or anything.

Reply:
Fixed with https://github.com/SUSE/metabroker/pull/1/commits/6e40ec03fc043577ac52e2cee27555281a773ca1.